src/components/cc-header-orga/cc-header-orga.js

```diff
@@ -3,185 +3,215 @@ import '../cc-img/cc-img.js';
           width: 100%;
         }
 
         .wrapper {
-          display: flex;
           overflow: hidden;
-          flex-wrap: wrap;
-          padding: 1em;
           border: 1px solid var(--cc-color-border-neutral, #aaa);
           background-color: var(--cc-color-bg-default, #fff);
           border-radius: var(--cc-border-radius-default, 0.25em);
-          gap: 1em;
         }
-        
-        :host([error]) .wrapper {
-          padding: 0;
-          border: none;
+
+        .header-body {
+          display: flex;
+          flex-wrap: wrap;
+          align-items: center;
+          justify-content: space-between;
+          padding: 1em;
+          gap: 1em;
         }
 
-        .wrapper.enterprise {
-          border-width: 2px;
-          border-color: var(--cc-color-bg-primary);
+        .identity {
+          display: flex;
+          align-items: center;
+          gap: 1em;
         }
 
-        .logo {
+        .logo,
+        .initials {
           width: 3.25em;
           height: 3.25em;
+          flex: 0 0 auto;
           border-radius: var(--cc-border-radius-default, 0.25em);
         }
 
-        .details,
-        .hotline {
+        .initials {
           display: flex;
-          flex-direction: column;
-          align-items: flex-start;
-        }
-
-        .details {
-          flex: 100 1 max-content;
+          align-items: center;
           justify-content: center;
-          row-gap: 0.2em;
+          background-color: var(--cc-color-bg-neutral);
         }
 
-        .hotline {
-          flex: 1 1 auto;
-          justify-content: space-between;
+        .initials span {
+          font-size: 0.85em;
         }
 
         .name {
-          font-size: 1.1em;
+          font-size: 1.3em;
           font-weight: bold;
         }
 
-        .hotline_number {
-          border-radius: 1em;
-        }
-
-        .hotline_number:focus,
-        .hotline_number:active {
-          outline: 0;
-        }
-
-        /* We can do this because we set a visible focus state */
-
-        .hotline_number::-moz-focus-inner {
-          border: 0;
+        .enterprise {
+          display: flex;
+          flex-direction: column;
+          gap: 0.5em;
         }
 
-        .hotline_number:focus {
-          outline: var(--cc-focus-outline, #000 solid 2px);
-          outline-offset: var(--cc-focus-outline-offset, 2px);
+        .enterprise-row {
+          display: flex;
+          gap: 0.5em;
         }
 
-        .hotline_number:hover {
-          box-shadow: 0 1px 3px rgb(0 0 0 / 40%);
-        }
+        .enterprise-row cc-icon {
+          --cc-icon-color: var(--cc-color-text-primary-highlight);
 
-        .hotline_number:active {
-          box-shadow: none;
+          flex: 0 0 auto;
         }
 
-        .hotline_number cc-badge {
-          /* Prevent space below badge because of text lines */
-          display: flex;
-          text-decoration: underline;
+        ::slotted([slot='footer']) {
```

Comment:
**Note:** I felt clever at the time but I'm not sure it's a good idea :laughing: 

We want the footer to have a style (grey background, same padding as the body for the content, a subtle top border).
But we only want this "footer" to be displayed when there is a slotted content.
If we set something like: 
```
<div class="footer">
  <slot name="footer"></slot>
</div>
```

The `div.footer` is always in the DOM and its style (border, padding, background) is visible even when there is no slotted content.
We could listen for slot changes to toggle a `hidden` attribute on the `div.footer` but I wanted to see if we could avoid this.

The current solution does work but it has some drawbacks: if, from outside this component, you define styles on the `<div slot="footer">`, they will override the styles defined here. It makes overriding the styles here very simple but maybe it's actually too easy and it also means you may override these by mistake :thinking:.

Reply:
I'm ok with this drawback as soon as it is explained in the @slot JSDoc section